luke-dvops/blog-app
# added test cases for Blog frontend

Changes Made
This pull request introduces a set of test cases for the blog frontend, focusing on key functionalities. The primary goal is to ensure the robustness and reliability of the frontend application.

Test Cases Overview

1. Login and Navigation
- Verify successful user login.
- Ensure proper navigation to the home page after login.

2. Create Post

- Validate the ability to create a new blog post.
- Check for correct navigation to the post details page after post creation.

3. Edit Post

- Confirm the functionality to edit an existing blog post.
- Check for the correct URL structure and content after the edit.

4. Delete Post

- Verify the ability to delete a blog post.
- Validate that the post is removed from the frontend and database.

Additional Information
The test suite uses [testing framework/library], [e.g., Mocha], and [assertion library], [e.g., Chai].
Selenium WebDriver is employed for browser automation.
The test environment is set up to interact with a test instance of the blog frontend hosted at [test URL].

How to Test
Clone this branch locally.
Install dependencies using [instruction, e.g., npm install].
Run the test suite using [instruction, e.g., npm test].
